File: static/game/GameClient/Objects/Figures/Figures.js
import {SkeletonArcher} from "./SkeletonArcher.js";
import {SkeletonWarrior} from "./SkeletonWarrior.js";
import {Peasant} from "./Peasant.js";
import {ElfMage} from "./ElfMage.js";
import {ElfAssassin} from "./ElfAssassin.js";
import {Spearman} from "./Spearman.js";

export class Figures{
    static get_figure(name){
        switch (name.toLowerCase()) {
            case "SkeletonArcher".toLowerCase(): return SkeletonArcher;
            case "Peasant".toLowerCase(): return Peasant;
            case "ElfMage".toLowerCase(): return ElfMage;
            case "ElfAssassin".toLowerCase(): return ElfAssassin;
            case "Spearman".toLowerCase(): return Spearman;
            case "SkeletonWarrior".toLowerCase(): return SkeletonWarrior;
            default: return null;
        }
    }
}

File: game/GameServer/Objects/Figures/Figures.py
from .SkeletonArcher import SkeletonArcher
from .Peasant import Peasant
from .ElfMage import ElfMage
from .ElfAssassin import ElfAssassin
from .Spearman import Spearman
from .SkeletonWarrior import SkeletonWarrior


class Figures:
    @staticmethod
    def get_figure(name: str):
        if name.lower() == "SkeletonArcher".lower():
            return SkeletonArcher
        elif name.lower() == "Peasant".lower():
            return Peasant
        elif name.lower() == "ElfMage".lower():
            return ElfMage
        elif name.lower() == "ElfAssassin".lower():
            return ElfAssassin
        elif name.lower() == "Spearman".lower():
            return Spearman
        elif name.lower() == "SkeletonWarrior".lower():
            return SkeletonWarrior
        return None
